examples/openai_embedding.py

```diff
@@ -0,0 +1,152 @@
+import argparse
+import os.path as osp
+from typing import List
+
+# Please run `pip install openai` to install the package
+import openai
+import torch
+import torch.nn.functional as F
+from torch import Tensor
+from tqdm import tqdm
+
+from torch_frame import stype
+from torch_frame.config.text_embedder import TextEmbedderConfig
+from torch_frame.data import DataLoader
+from torch_frame.datasets import AmazonFineFoodReviews
+from torch_frame.nn import (
+    EmbeddingEncoder,
+    FTTransformer,
+    LinearEmbeddingEncoder,
+    LinearEncoder,
+)
+
+# Please run `export OPENAI_API_KEY=...` before running the code
+# Notice that there are 568,454 rows and 2 text columns, it will
+# cost some money to get the text embeddings by using OpenAI API
```

Comment:
Is it possible to supply it as an argument of the script. You can first check if it's supplied, then if it's in env var. If not, you error out.

Reply:
Sure